examples/callback-guest/callback-guest.rs

```diff
@@ -4,7 +4,6 @@ extern "C" {
     fn host_callback() -> u32;
 }
 
-#[no_mangle]
```

Comment:
What's the motivation for this change?

Reply:
Oh that was not supposed to be part of this PR -- I was testing something for someone on our public slack